packages/bitcore-client/bin/wallet-derive-address

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const program = require('../ts_build/program');
+const { Wallet } = require('../ts_build/wallet');
+const promptly = require('promptly');
+
+program
+  .version(require('../package.json').version)
+  .option('--name <name>', 'REQUIRED - Wallet Name')
+  .option('--change [change]', 'optional - should derive change addresses')
+  .option('--addressIndexes [addressIndexes]', 'optional - takes one or more indexes')
+  .option('--path [path]', 'optional - Custom wallet storage path')
+  .option('--storageType [storageType]', 'optional - name of the database to use (default Level)')
+  .parse(process.argv);
+
+const main = async () => {
+  const { name, path, storageType, addressIndexes } = program;
+  let { change } = program;
+  change = (change === 'true');
```

Comment:
it looks like destructuring it will not result change to boolean true/false, which has it be boolean. Another way that I can think of is to check if change is defined to avoid the program to crash and do `JSON.parse(change)`.

Reply:
Can I have anyone interested in buying bitcore